Header header
#
# each of the members is either Y or N, to indicate
# whether the robot currently reports and active
# collission condition
#
string forwardCollision
string backwardCollision

